### PR TITLE
Fix geodesic s3fs docs

### PR DESCRIPTION
**Pull request by osterman**
## what
* Clear up documentation relating to `s3fs`

## why
* Previous version was incompleted

## references
- https://github.com/cloudposse/docs/issues/123

### CODE REVIEW

File: content/geodesic/s3fs.md [hunk 1]
```diff
@@ -4,5 +4,43 @@ description: "Geodesic provides an easy way for mounting encrypted S3 buckets to
 ---
 
 The geodesic base image ships with a number of utility scripts:
-- [`s3`](https://github.com/cloudposse/geodesic/blob/master/rootfs/usr/local/bin/s3) - for managing S3 filesystems in `/etc/fstab`
-- [`s3fs`] - a thin wrapper around `goofys` for mounting S3 filesystems with a local cache in `/dev/shm`
+
+- [`/usr/local/bin/s3`](https://github.com/cloudposse/geodesic/blob/master/rootfs/usr/local/bin/s3) - makes it easier to manage S3 filesystems in `/etc/fstab`
+- [`/usr/local/bin/s3fs`](https://github.com/cloudposse/geodesic/blob/master/rootfs/usr/local/bin/s3fs) - a thin wrapper around `goofys` for mounting S3 filesystems with a local cache in `/dev/shm`. It's the command executed by `mount` (e.g. `s3fs#/
```
Comment (aknysh) [inline content/geodesic/s3fs.md]: (e.g. `s3fs#/`) ?

File: content/geodesic/s3fs.md [hunk 2]
```diff
@@ -4,5 +4,43 @@ description: "Geodesic provides an easy way for mounting encrypted S3 buckets to
...
+
+# Use-cases
+
+A few common use-cases have arrisen for mounting S3 buckets inside of the geodesic shell.
+
+1. **Master SSH Keys**. When using [terraform-aws-keypair](https://github.com/cloudposse/terraform-aws-key-pair) to generate a set of SSH keys, those keys need to be securely stored somewhere. An private, versioned, encrypted S3 bucket is as good a place as any. The [terraform-aws-tfstate-backend](https://github.com/cloudposse/terraform-aws-tfstate-backend) provide a bucket suitable for this purpose.
```
Comment (aknysh) [inline content/geodesic/s3fs.md]: [terraform-aws-key-pair]

Comment (aknysh) [reply content/geodesic/s3fs.md]: A private, versioned

File: content/geodesic/s3fs.md [hunk 3]
```diff
@@ -4,5 +4,43 @@ description: "Geodesic provides an easy way for mounting encrypted S3 buckets to
...
+{{% dialog type="info" icon="fa fa-info-circle" title="Note" %}}
+The `${TF_BUCKET}` is evaluated (interpolated) at runtime by the `s3fs` command called by `mount`. This allows it to be highly dynamic. Under certain circumstances this may not be wanted. By removing the single quotes around `'${TF_BUCKET}'` in the `s3 fstab` command, it will be evaluated at `docker build` time rather than at run time.
+{{% /dialog %}}
+
+# Mounting Buckets
+
+To mount buckets, just run `mount -a` after having assumed roles. This is the same as calling `s3 mount`.
+
+{{% dialog type="info" icon="fa fa-info-circle" title="Note" %}}
+The `init-terraform` command calls `s3 mount` this automatically.
```
Comment (aknysh) [inline content/geodesic/s3fs.md]: The `init-terraform` command calls `s3 mount` automatically

File: content/glossary/goofys.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+description: "Goofys is a utility that implements S3-backed filesystems using FUSE."
+terms:
+- goofys
+tags:
+- aws
+- s3
+- s3fs
+- fuse
+---
+Learn more about how to use [`goofys`]({{< relref \"tools/goofys.md\" >}}) and [how we use it]({{< relref "geodesic/s3fs.md" >}}) with geodeisc.
```
Comment (aknysh) [inline content/glossary/goofys.md]: do we need a backslash `\` here?
```
\"tools/goofys.md\" >
```

File: content/glossary/interpolation.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: "String Interpolation"
+description: '[String Interpolation](https://en.wikipedia.org/wiki/String_interpolation) is the process of evaluating a string  containing one or more placeholders (e.g. `$FOOBAR` or `{{...}}`) and replacing the placeholders with their corresponding values.'
```
Comment (aknysh) [inline content/glossary/interpolation.md]: why `'` (single quotes) around the description?

File: content/geodesic/s3fs.md (hunk 1 above)
Comment (osterman) [reply content/geodesic/s3fs.md]: oops

**Pull request merged**